lang/cs/StyleCop.MSBuild.Targets

```diff
@@ -0,0 +1,142 @@
+﻿<!--
```

Comment:
This file is from [StyleCop.MSBuild project](https://bitbucket.org/adamralph/stylecop-msbuild/src/db5a8c8167e7fafbc5610f6976edadb630c9cca4/build/StyleCop.MSBuild.Targets?at=default&fileviewer=file-view-default). The project is MIT Licensed.